google/cloud/sql/connector/connector.py

```diff
@@ -229,3 +243,169 @@ def connect(instance_connection_string: str, driver: str, **kwargs: Any) -> Any:
     if _default_connector is None:
         _default_connector = Connector()
     return _default_connector.connect(instance_connection_string, driver, **kwargs)
+
+
+async def get_connection(
+    icm: InstanceConnectionManager,
+    connect_function: Callable,
+    ip_type: IPTypes,
+    loop: asyncio.AbstractEventLoop,
+    **kwargs: Any,
+) -> Any:
```

Comment:
I don't really understand this function's signature (or why we need it). It's also very short (minus the docstring). Are we sure we shouldn't inline here?